src/js/components/RangeInput/RangeInput.js

```diff
@@ -102,7 +102,7 @@ const RangeInput = forwardRef(
         aria-label={a11yTitle}
         aria-valuemax={max}
         aria-valuemin={min}
-        aria-valuenow={value}
+        aria-valuenow={value !== undefined && value !== '' ? value : undefined}
```

Comment:
If value is undefined, wouldn't `value !== undefined && value !== '' ? value : undefined` and `value !== '' ? value : undefined` result in the same behavior? Just wanting to understand if we actually need the `value !== undefined` check

Reply:
yes you are correct just simplified